core/src/lib.rs

```diff
@@ -291,6 +291,45 @@ impl Schema {
     pub fn identifies_attribute(&self, x: &NamespacedKeyword) -> bool {
         self.get_entid(x).map(|e| self.is_attribute(e)).unwrap_or(false)
     }
+
+    pub fn as_edn_value(&self) -> edn::Value {
+        let mut s = "[ ".to_string(); 
+        for (entid, attribute) in &self.schema_map {
+            s.push_str("{");
+            s.push_str(&format!(" :db/id :{:?}", entid));
```

Comment:
ahh. I think I got wrong-footed by this link in the original bug [https://github.com/mozilla/mentat/blob/c585715224c180ff84630c05aa763d0d271f350a/db/src/bootstrap.rs#L86](https://github.com/mozilla/mentat/blob/c585715224c180ff84630c05aa763d0d271f350a/db/src/bootstrap.rs#L86)